### ISSUE TITLE
Inner Section height

### ISSUE THREAD

**Issue by twiw** [dudaster/ele-custom-skin#136]
Hey I've played with this a lot and finally figured out something. Something about this is messing up my spacing, mostly in inner sections. Here's a template I pulled from elementor (yes I've tried several others, they all do the same thing). When I had ele custom skin activated, it does this: 
![Annotation 2020-01-10 1000342](https://user-images.githubusercontent.com/50600620/72171951-34f57f00-3391-11ea-8142-d530e60b8d98.png)

When I deactivated it, it does this.
![Annotation 2020-01-10 100034](https://user-images.githubusercontent.com/50600620/72171976-4048aa80-3391-11ea-9ba4-75d2b4e30a11.png)

I also tried a custom footer on my own and couldn't even build it because it had so much height. It has happened in several other places on my site too. I can't find where the space is coming from in css when I enter the console! That's the weird part. Anyone else had this problem?

**Comment by dudaster** [comment]
Can you provide an url with ECS activated?

**Comment by twiw** [comment]
No, it's a local site, sorry about that. I'll get it up in a few days and
hide it, then I can send you a temp login. Thanks.
